src/map/zone.h

```diff
@@ -318,7 +318,7 @@ enum ZONEID : uint16
     ZONE_DHO_GATES                      = 272,
     ZONE_WOH_GATES                      = 273,
     ZONE_OUTER_RAKAZNAR                 = 274,
-    ZONE_OUTER_RAKAZNAR_U               = 275,
+    ZONE_OUTER_RAKAZNAR_U1              = 275,
```

Comment:
don't forget Throne Room [V] at 229, you got in the lua side but not the cpp header